src/foam/nanos/crunch/CapabilityIsVisible.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'CapabilityIsVisible',
+  extends: 'foam.mlang.predicate.AbstractPredicate',
+  implements: [ 'foam.core.Serializable' ],
+
```

Comment:
Could add:
axioms: [ { class: 'foam.pattern.Singleton' } ],
to make more efficient.

Reply:
@kgrgreer added